src/qt/sendcoinsdialog.cpp

```diff
@@ -889,17 +885,29 @@ void SendCoinsDialog::coinControlUpdateLabels()
     }
 }
 
-SendConfirmationDialog::SendConfirmationDialog(const QString& title, const QString& text, const QString& informative_text, const QString& detailed_text, int _secDelay, QWidget* parent)
-    : QMessageBox(parent), secDelay(_secDelay)
+SendConfirmationDialog::SendConfirmationDialog(const QMessageBox::Icon icon, const QString& title, const QString& text, const QMessageBox::StandardButton yes_button, const QString &yes_button_text, const QMessageBox::StandardButton cancel_button, const QString& informative_text, const QString& detailed_text, int _secDelay, QWidget *parent)
+    : QMessageBox(parent),
+    m_yes_button_text(yes_button_text),
+    secDelay(_secDelay)
 {
-    setIcon(QMessageBox::Question);
+    setIcon(icon);
```

Comment:
Instead of another constructor argument, let the caller change the icon? Like
```cpp
SendConfirmationDialog confirmationDialog(tr("Confirm fee bump"), ...);
confirmationDialog.setIcon(QMessageBox::Question);

Reply:
No reply?

Reply:
I don't know that it makes sense to create a SCD without a chosen icon...

Reply:
I'd be happy to have `QMessageBox::Question` by default, and the caller is free to change.

Reply:
Ok, done